tests/unit/test_charm.py

```diff
@@ -11,7 +11,7 @@
 
 logger = logging.getLogger(__name__)
 
-LOG_FILE = "/var/log/openfga-k8s"
+LOG_FILE = "/openfga-k8s.log"
```

Comment:
random question: what's the reason behind changing the path?

Reply:
In this unit test the misalignment between this and the implementation caused a failure.
In the main implementation the path was stripped from the "/var/log" because that path didn't exist, and we preferred not to complicate the implementation more.
I openend this [issue](https://github.com/canonical/openfga-rock/issues/4) to keep track of this in the openfga-rock project.